fixit/cli/add_new_rule.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in thefla
+# LICENSE file in the root directory of this source tree.
+
+# Usage:
+#
+#   $ python -m fixit.cli.add_new_rule --help
+#   $ python -m fixit.cli.add_new_rule
+#   $ python -m fixit.cli.add_new_rule --path fixit/rules/new_rule.py
+
+import argparse
+from pathlib import Path
+
+from libcst.codemod._cli import invoke_formatter
+
+from fixit.common.config import get_lint_config
+
+
+_LICENCE = """\
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+
+_IMPORTS = """
```

Comment:
Just my opinion, but wouldn't it be cool to have `libcst as cst` and `libcst.matchers as m` imports here as well?

Reply:
Will add those imports.

Reply:
Done